docs/adr/AsynkronKoeForBrevBestillinger.md

```diff
@@ -0,0 +1,39 @@
+# Asynkron kø for brevbestillinger
+
+* Status: [draft]
+* Deciders: Håkon Heggholmen
+
+Technical Story: [Lage asynkron kø for brevbestillinger]
+
+## Context and Problem Statement
+
+I dag har vi kun ett synkront løp for bestilling av brev. Det å rendre en PDF med LaTeX tar flere sekunder, og vi har 
+begrenset med ledig synkron kapasitet til en hver tid. For å komme rundt dette problemet i dag må konsumenter hensynta
+våres ledige kapasitet. Konsumenter må også ta stilling til tekniske feil som team pensjonsbrev er ansvarlige for å løse
+, og ha mekanismer for å forsøke på nytt i disse tilfellene. Det koster også penger å alltid ha tilgjengelige instanser 
+kjørende for å kunne håndtere store volum synkrone bestillinger.
+
+## Decision Drivers
+* [Ekstra kompleksitet i løsningene til konsumenter]
+* [Forsinkelser i bestillingsløpet til konsumenter]
+* [Uavklart ansvarsforhold ved tekniske feilmeldinger fra brevløsningen]
+* [Kostnader ved ubrukt reservert CPU kapasitet på NAIS]
+
+## Considered Options
+
+* [option 1] Lage en kafka kø mellom brevbaker og pdf-bygger
+* [option 2] 
```

Comment:
Vi bør også ha 0-alternativet som ei moglegheit, synes eg.

Reply:
Tror jeg omformulerer det litt så det blir pros and cons for hele beslutningen, og ikke bare options. Når jeg lå til redis som option ble det naturlig å legge til pros and cons i forhold til de ulike valgene.